### ISSUE TITLE
Truncate geometry field

### ISSUE THREAD

**Issue by CharliePatterson** [digital-land/submit#440]
## Context:

- On the issues details page when a user is viewing a single record / entity from a geospatial dataset, the geometry field can sometimes be very long, expanding the length of the page substantially.
- We should truncate the geometry field so that we're not showing the full entry.

![Image](https://github.com/user-attachments/assets/c8d967af-1991-4fae-8c6b-1c51cd256da8)

- @alextea what's the design steer on this?

**Comment by alextea** [comment]
Yes this should be truncated. Possibly with an option to expand the whole value if it's useful. Something to look into.

**Comment by neilfwar** [comment]
Nice to have - TBC
